gensim/corpora/wikicorpus.py

```diff
@@ -576,7 +576,7 @@ class WikiCorpus(TextCorpus):
     """
     def __init__(self, fname, processes=None, lemmatize=utils.has_pattern(), dictionary=None,
                  filter_namespaces=('0',), tokenizer_func=tokenize, article_min_tokens=ARTICLE_MIN_WORDS,
-                 token_min_len=TOKEN_MIN_LEN, token_max_len=TOKEN_MAX_LEN, lower=True, filter_articles=None):
+                 token_min_len=TOKEN_MIN_LEN, token_max_len=TOKEN_MAX_LEN, lower=True, filter_articles=None, metadata=False):
```

Comment:
All parameters must be documented.

Also, this will probably fail the code style check (line too long).

Reply:
Okay, closing this pull request and creating another after making required changes.

Reply:
You could have applied the changes to this PR. Pull requests are editable.